game_api.txt

```diff
@@ -395,3 +395,31 @@ default.grow_new_pine_tree(pos)
 
 default.grow_new_acacia_tree(pos)
 ^ Grows a new design acacia tree at pos
+
+#Tnt API
```

Comment:
TNT is not a sub section of Trees, you should put it here: https://github.com/red-001/minetest_game/blob/tnt-api/game_api.txt#L147